api/room/schema_query.py

```diff
@@ -25,6 +26,15 @@ class RatiosPerRoom(graphene.ObjectType):
     ratios = graphene.List(RatioOfCheckinsAndCancellations)
 
 
+class RemoteRooms(graphene.ObjectType):
```

Comment:
Great work @kampkelly.

My suggestion is to use RemoteRoom here since you're dealing with one object of sorts.

Reply:
Thanks @RonKbS . This has been done.